shared/charts/couchdb/templates/statefulset.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: {{ template "couchdb.fullname" . }}
   labels:
     app: {{ template "couchdb.name" . }}
-    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```

Comment:
I think I understand why you are changing this. Are there any consequences of this change (e.g. users of this chart are no longer able to deploy multiple CouchDBs to a given cluster, or something)?

Possibly related question: can this change be contributed back to upstream? If so, please do so before closing this ticket :). If not, I'd like to discuss the situation further.

Reply:
I'm not aware of any negative consequences of this, Helm does no relly on this info. AFAIK the only result is that you'll be actually allowed to update the DB (as opposed to thinking it's been updated and Helm/TF silently ignoring that real state is not desired state, which I think is much more dangerous behavior - think about the security update we did to CouchDB recently).